### ISSUE TITLE
floating point exception in lora train Wan2.1  i2v -14B

### ISSUE THREAD

**Issue by sunhe97** [modelscope/DiffSynth-Studio#591]
Error： 

initializing deepspeed distributed: GLOBAL_RANK: 3, MEMBER: 4/8
initializing deepspeed distributed: GLOBAL_RANK: 6, MEMBER: 7/8
Enabling DeepSpeed BF16. Model parameters and inputs will be cast to `bfloat16`.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0,1,2,3,4,5,6,7]
LOCAL_RANK: 6 - CUDA_VISIBLE_DEVICES: [0,1,2,3,4,5,6,7]
LOCAL_RANK: 3 - CUDA_VISIBLE_DEVICES: [0,1,2,3,4,5,6,7]
LOCAL_RANK: 7 - CUDA_VISIBLE_DEVICES: [0,1,2,3,4,5,6,7]
LOCAL_RANK: 1 - CUDA_VISIBLE_DEVICES: [0,1,2,3,4,5,6,7]
LOCAL_RANK: 4 - CUDA_VISIBLE_DEVICES: [0,1,2,3,4,5,6,7]
LOCAL_RANK: 5 - CUDA_VISIBLE_DEVICES: [0,1,2,3,4,5,6,7]
LOCAL_RANK: 2 - CUDA_VISIBLE_DEVICES: [0,1,2,3,4,5,6,7]

  | Name | Type             | Params | Mode
-------------------------------------------------
0 | pipe | WanVideoPipeline | 16.5 B | eval
-------------------------------------------------
76.7 M    Trainable params
16.4 B    Non-trainable params
16.5 B    Total params
65,886.224Total estimated model params size (MB)
5144      Modules in train mode
1         Modules in eval mode
/opt/conda/lib/python3.10/site-packages/lightning/pytorch/trainer/connectors/data_connector.py:425: The 'train_dataloader' does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` to `num_workers=23` in the `DataLoader` to improve performance.
Epoch 0:   0%|                                                                                                                                                       | 0/63 [00:00<?, ?it/s]Floating point exception (core dumped)

use 8 * H20 to do lora train，video is 7s，Is it too long？

train command in terminal：

python examples/wanvideo/train_wan_t2v.py --task train --train_architecture lora --dataset_path data/example_dataset --output_path ./models --dit_path "models/Wan2.1-I2V-14B-480P/diffusion_pytorch_model-00001-of-00007.safetensors,models/Wan2.1-I2V-14B-480P/diffusion_pytorch_model-00002-of-00007.safetensors,models/Wan2.1-I2V-14B-480P/diffusion_pytorch_model-00003-of-00007.safetensors,models/Wan2.1-I2V-14B-480P/diffusion_pytorch_model-00004-of-00007.safetensors,models/Wan2.1-I2V-14B-480P/diffusion_pytorch_model-00005-of-00007.safetensors,models/Wan2.1-I2V-14B-480P/diffusion_pytorch_model-00006-of-00007.safetensors,models/Wan2.1-I2V-14B-480P/diffusion_pytorch_model-00007-of-00007.safetensors" --steps_per_epoch 500 --max_epochs 10  --learning_rate 1e-4 --lora_rank 16 --lora_alpha 16 --lora_target_modules "q,k,v,o,ffn.0,ffn.2" --accumulate_grad_batches 1 --use_gradient_checkpointing --image_encoder_path "models/Wan2.1-I2V-14B-480P/models_clip_open-clip-xlm-roberta-large-vit-huge-14.pth" --training_strategy deepspeed_stage_2

**Comment by sunhe97** [comment]
solution：https://blog.csdn.net/m0_46435741/article/details/146028843?utm_medium=distribute.pc_relevant.none-task-blog-2~default~baidujs_baidulandingword~default-0-146028843-blog-146531075.235^v43^control&spm=1001.2101.3001.4242.1&utm_relevant_index=2
